vip_spec.xsd

```diff
@@ -337,8 +340,11 @@ reference elements for polling location
         <xs:element name="Person">
           <xs:complexType>
             <xs:sequence>
+              <!-- SURVEY: Types xs:string and InternationalizedText seem
+                   to be used inconsistently here. Suggestions?
```

Comment:
I thought about raising that with 1622.2, but as a latecomer, and in getting the impression that they are really trying to wrap up the process and only want serious issues, I didn't saying anything.